config.toml

```diff
@@ -95,39 +95,40 @@ js = [
 ]
 
 [[params.versions]]
-fullversion = "v1.18.0"
+fullversion = "v1.19.0"
+version = "v1.19"
+githubbranch = "v1.19.0"
+docsbranch = "master"
+url = "https://kubernetes.io"
+
+[[params.versions]]
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "v1.18.0"
+githubbranch = "v1.18.8"
 docsbranch = "release-1.18"
-url = "https://kubernetes.io"
+url = "https://v-18.docs.kubernetes.io"
```

Comment:
```suggestion
url = "https://v1-18.docs.kubernetes.io"
```
I think

Reply:
OMG. I can't believe I am making mistakes on the same URL twice 🙈 . Fix coming up right away.

Reply:
I hope I fixed it right this time.